triton-vm/src/lib.rs

```diff
@@ -568,7 +572,11 @@ mod tests {
         #[strategy(arb())] hash_preimage: Digest,
         #[strategy(arb())] some_tie_to_an_outer_context: Digest,
     ) {
-        let hash_digest = hash_preimage.hash::<Tip5>().values();
+        let hash_digest = Tip5::hash_pair(
+            hash_preimage,
+            Digest::new([BFieldElement::zero(); DIGEST_LENGTH]),
```

Comment:
Isn't this just
```suggestion
            Digest::default(),
```
?

Alternatively, if it is not:
```suggestion
            Digest::new(bfe_array![0; DIGEST_LENGTH]),
```

Reply:
It is just `Digest::default()`. I wasn't sure how comfortable using `default` for this value. I'm OK with it though.

Reply:
I would say since this is a test, it's fine; If something about `Digest::default()` changes, it will be spotted here immediately.